### DISCUSSION TITLE
Instalei o Tailwind da forma que a documentação pede mas na hora de utlizar ele no documento html, nao gera mudanças na pagina web

### DISCUSSION THREAD

**Discussion by Sottaaa** [tailwindlabs/tailwindcss#11492] [Help]
Fiz todo o passo a passo que esta na documentação, instalei via npm, iniciei com o npx, coloquei a configuração base no tailwind.config.js , criei a pasta src com o input.css nela e copiei o tailwind para a pagina, depois usei o comando para criar a pasta dist com o output nela e linkei no header no html com <link rel="stylesheet" href="dist/output.css">. Mesmo apos seguir todo passo a passo, o tailwind nao esta sendo interpretado pelo navegador, consigo ler a classe adicionada ao elemento pelo inspecionar elemento, mas nao atribui nenhum efeito visual ao elemento em si

**Comment by Sethi237** [comment]
Sure, I can help you with that.

There are a few things that could be causing this issue.

- You may not have installed Tailwind correctly. Make sure that you have installed Tailwind globally with the following command:

Code snippet
npm install -g tailwindcss

- You may not have linked the Tailwind CSS file to your HTML file. Make sure that you have added the following line to the header of your HTML file:

Code snippet
<link href="tailwind.css" rel="stylesheet">

- You may have a typo in your Tailwind configuration file.** Make sure that you have checked your Tailwind configuration file for any typos.

- You may be using a browser that does not support Tailwind.** Tailwind is supported by most modern browsers, but there are a few browsers that do not support it. If you are using one of these browsers, you may need to use a different CSS framework.

If you have checked all of these things and you are still having trouble, you can try the following:

- Restart your browser.** This may clear up any temporary issues that are causing the problem.
- Clear your browser's cache.** This will remove any old or corrupt files that may be causing the problem.

Try a different browser. If you are using a browser that does not support Tailwind, you can try a different browser that does support it.

I hope this helps!     [marykayintouch](url)
